app/assets/stylesheets/v3/layout_components/sections.scss

```diff
@@ -1,71 +1,65 @@
 .heading {
-  padding: $base-padding*0.75 $base-padding;
+  padding: $base-padding $base-padding $base-padding*2;
   color: $white;
 }
 
 .section {
-  margin-bottom: $base-margin;
+  margin-bottom: $base-margin*3;
 
   &__heading {
     @extend .heading;
-    background: $brand-blue-light;
+    background: transparent;
+    color: $dark-grey-text;
 
     &--no-bg {
       padding-bottom: $base-padding;
-      color: $charcoal;
+      color: $dark-grey-text;
     }
 
     &--no-padding {
       background: $brand-blue-light;
-      color: $white;
+      color: $dark-grey-text;
     }
 
     &--grey-bg {
-      @extend .heading;
-      background: $slate;
+      @extend .section__heading;
```

Comment:
not sure if my comment above would make this not work as intended anymore, but we should probably try to find a place this is used to check.

Reply:
not sure if this is being used. I think we should leave it in just in case, and maybe for phase 2 see if we can remove it. Worst case it will just show up with a transparent bg, but with the right font sizes for a heading.